apps/test-suite/tests/NewNotifications.js

```diff
@@ -1179,6 +1217,69 @@ export async function test(t) {
         10000
       );
     });
+
+    onlyInteractiveDescribe(
+      'triggers a repeating daily notification. only first scheduled event is awaited and verified.',
+      () => {
+        let timesSpyHasBeenCalled = 0;
+        const identifier = 'test-scheduled-notification';
+        const notification = {
+          title: 'Scheduled notification',
+          data: { key: 'value' },
+          badge: 2,
+          vibrate: [100, 100, 100, 100, 100, 100],
+          color: '#FF0000',
+        };
+
+        t.beforeEach(async () => {
+          Notifications.setNotificationHandler({
+            handleNotification: async () => {
+              timesSpyHasBeenCalled += 1;
+              return {
+                shouldShowAlert: false,
+              };
+            },
+          });
+        });
+
+        t.afterEach(() => {
+          Notifications.setNotificationHandler(null);
+          Notifications.cancelAllScheduledNotificationsAsync();
+        });
+
+        t.it(
+          'triggers a repeating daily notification. only first event is verified.',
+          async () => {
+            const dateIn1Mimutes = new Date(
+              new Date().getTime() + (Platform.OS === 'ios' ? 120000 : 60000)
```

Comment:
On iOS because we are using the calendar with repeat, it needs to be greater than 60 seconds.